clarify and expand TEE manufacturer assets (#26)
patcg/docs-and-reports: threat-model/readme.md

#### 1.9.1 Assets

1. While TEE Manufacturers take measures to prevent the leakage of private keys to themselves, we assume that they may have access to these private keys embedded in the TEEs.
2. The corresponding public keys for a specific TEE.

> 3. The "root" key used to verify the authenticity of a specific public key.

Reviewer: This might instead be:

```suggestion
3. The "root" private key used to authenticate the keys used in each TEE.
```

The "root" public key can be assumed as public knowledge and not worth mentioning here.  It wasn't clear which you meant here, verification being something that only depends on the public key.
